src/raw.imageio/rawinput.cpp

```diff
@@ -453,6 +453,14 @@ RawInput::open_raw(bool unpack, const std::string& name,
     // Output 16 bit images
     m_processor->imgdata.params.output_bps = 16;
 
+    // exposing max_raw_memory_mb setting
+    static_assert(sizeof(m_processor->imgdata.rawparams.max_raw_memory_mb)
+                      == sizeof(int),
+                  "Sizes must match");
+    config.getattribute("raw:max_raw_memory_mb", TypeDesc::INT,
+                        &m_processor->imgdata.rawparams.max_raw_memory_mb);
+
+
```

Comment:
Might it be simpler to express this more like

    m_processor->imgdata.rawparams.max_raw_memory_mb = 
        config.get_int_attribute("raw:max_raw_memory_mb", <default value>);

?? Then you wouldn't need the static assert, and various type conversions would happen automatically if the types ever change.

Also, would you mind please documenting the existence and meaning of this attribute in the raw section of builtinplugins.rst ? Thanks.

Reply:
What's a good default for this? The other reason I like my suggestion above is that it's probably a good idea to by default impose some reasonable limit, even if this parameter is not passed from the user. We shouldn't allow big files to consume unlimited memory in libraw unless specifically asked to do so.

Reply:
Default in libraw is 2048 (2GB)
https://github.com/LibRaw/LibRaw/blob/9bcb8a1d9593ba67e4eb67fed716efc5e1353d5c/libraw/libraw_const.h#L24
and
https://github.com/LibRaw/LibRaw/blob/9bcb8a1d9593ba67e4eb67fed716efc5e1353d5c/doc/API-datastruct.html#L556-L558
Libraw itself doesn't have a upper limit.

Reply:
Great, let's just express that default as I suggested above, then you don't need the static_assert or the needing pointers.